overlord/snapstate/autorefresh.go

```diff
@@ -292,7 +293,9 @@ func (m *autoRefresh) Ensure() error {
 		}
 
 		err = m.launchAutoRefresh()
-		m.nextRefresh = time.Time{}
+		if _, ok := err.(*httputil.UnretriedNetworkError); !ok {
+			m.nextRefresh = time.Time{}
+		} // else - refresh will be retried after refreshRetryDelay
```

Comment:
we probably want to reduce again a bit the refreshRetryDelay to 20 min

Reply:
Done.